### PR TITLE
Updated SAML/SSO documentation

### PR DESCRIPTION
**Pull request by hendrik-hackerone**

### CODE REVIEW

**Review by DeRRudi77 — APPROVED** [review]

File: docs/organizations/single-sign-on-sso-via-saml.md [hunk 1]
```diff
@@ -21,112 +21,121 @@ HackerOne supports Single Sign-On (SSO) through Security Assertion Markup Langua
 ### Set Up
 
 To configure Single Sign-On via SAML:
-1. Go to **Program Settings > General > Authentication**.
+1. Go to **Organization Settings > Authentication > SAML (SSO)**.
 
-><i>Note: You must have User Management permissions to setup SAML.</i>
+><i>Note: You must be organization administrator to setup SAML.</i>
```
[the remark this answers is not in the document]
Comment (hendrik-hackerone) [reply docs/organizations/single-sign-on-sso-via-saml.md]: 👍

File: docs/organizations/single-sign-on-sso-via-saml.md [hunk 2]
```diff
@@ -21,112 +21,121 @@
...
+
+![SAML setup 10](./images/saml-sso-setup-8.png)
+
+3. Click **configure** next to *X509 ALTERNATIVE CERTIFICATE* 
 
-![Authentication settings page with SAML configured](./images/alt-certificate-1.png)
+![SAML setup 11](./images/saml-sso-setup-9.png)
 
-3. Enter the alternative certificate in the **Configure alternative certificate** window.
+5. Enter the alternative certificate in the **Configure alternative certificate** window.
```
[the remark this answers is not in the document]
Comment (hendrik-hackerone) [reply docs/organizations/single-sign-on-sso-via-saml.md]: 👍

File: docs/organizations/sso-jit.md [hunk 3]
```diff
@@ -22,23 +22,5 @@ Option Type | Details
...
+Advanced | *Only for Enterprise programs* <br><br>Enables organizations to control membership and permission level from their SSO provider. When configured, the attributes for the users membership and group will be used to assign the user to your program and the appropriate group in HackerOne with the associated permissions. You can confirm the memberships are being added properly by viewing your program [audit log](audit-logs.html).<br><br>To configure this provisioning, contact support@hackerone.com after your SAML configuration is enabled and HackerOne will assist in the setup.
```
[the remark this answers is not in the document]
Comment (hendrik-hackerone) [reply docs/organizations/sso-jit.md]: 👍

**Comment by hendrik-hackerone** [comment]
> I added a few nitpicky comments, but also the step order might be wrong in one place that I pointed out.
> 
> Image of Dallas **Dallas**
> 
> Reviewed with ❤️ by [PullRequest](https://www.pullrequest.com)

Thanks for your suggestions!

**Pull request merged**